packages/fxa-content-server/app/scripts/templates/settings/account_recovery/confirm_password.mustache

```diff
@@ -17,7 +17,7 @@
 
             <div class="button-row">
                 <button class="primary-button generate-key-link">{{#t}}Continue{{/t}}</button>
-                <button class="button-link cancel-link">{{#t}}Cancel{{/t}}</button>
+                <button type="button" class="button-link cancel-link">{{#t}}Cancel{{/t}}</button>
```

Comment:
Buttons receives `type="button"` by default so we shouldn't need to add this, I checked into your branch and took this out and your solution still works without it. :)

Reply:
You're right!